R/simulation-run-options.R

```diff
@@ -0,0 +1,27 @@
+#' @title SimulationRunOptions
+#' @docType class
+#' @description  Options to be passed to the simulation engine
+#' @field numberOfCoresToUse (Maximal) number of cores to be used (1 per default)

```

Comment:
Some kind of meta configuration of the simulation run. This will be updated with more parameters if needed